lib/logstash/outputs/scalyr.rb

```diff
@@ -562,6 +571,7 @@ def send_status
 
     if !@last_status_transmit_time
       status_event[:attrs]['message'] = "Started Scalyr LogStash output plugin."
+      status_event[:attrs]['serverHost'] = Socket.gethostname
```

Comment:
Not sure the cost of this call.  Should we call `Socket.gethostname` once and cache the value instead of every time we send the status?